cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorUtils.java

```diff
@@ -28,10 +38,42 @@
   public static final String ANOMALY_DETECTION_TIME_MS_OBJECT_CONFIG = "anomaly.detection.time.ms.object";
   public static final long MAX_METADATA_WAIT_MS = 60000L;
   public static final Anomaly SHUTDOWN_ANOMALY = new BrokerFailures();
+  public static final Random RANDOM = new Random();
 
   private AnomalyDetectorUtils() {
   }
 
+  /**
+   * Create a Kafka consumer for retrieving reported Maintenance plans.
+   * The consumer uses {@link String} for keys and {@link MaintenancePlan} for values.
+   *
+   * @param configs The configurations for Cruise Control.
+   * @return A new Kafka consumer
+   */
+  public static Consumer<String, MaintenancePlan> createMaintenanceEventConsumer(Map<String, ?> configs, String groupIdPrefix) {
+    // Get bootstrap servers
+    String bootstrapServers = configs.get(MonitorConfig.BOOTSTRAP_SERVERS_CONFIG).toString();
+    // Trim the brackets in List's String representation.
```

Comment:
The output of this method is not guaranteed to be stable across Java versions.

Reply:
ArrayList.toString() has a specification, but that's not true for all implementations.

Reply:
I believe we discussed a similar issue under a different PR: https://github.com/linkedin/cruise-control/pull/1223#discussion_r433685111

It is true that the `List` interface itself does not enforce a specific `toString()` response format, but I don't suppose we expect to use / support collections that are not based on the standard https://docs.oracle.com/javase/7/docs/api/java/util/AbstractCollection.html#toString(), whose openJDK implementation for `toString()` hasn't changed from 8 to 11?

Reply:
The previous code review was based on emitting the output in log messages.  I guess it's fine, but it seems strange because it's already a List it's possible to do the following in single line.
```java
  list.stream().collect(Collectors.joining( "," ) )
```

Reply:
I propose the following, which simplifies the statement w/o using the stream API.

```
    String bootstrapServers = String.join(",", (List<String>) configs.get(MonitorConfig.BOOTSTRAP_SERVERS_CONFIG));
```